static/app/views/detectors/components/issuesList.tsx

```diff
@@ -0,0 +1,135 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import AssigneeSelectorDropdown from 'sentry/components/assigneeSelectorDropdown';
+import GridEditable, {type GridColumnOrder} from 'sentry/components/gridEditable';
+import {Grid} from 'sentry/components/gridEditable/styles';
+import {GroupSummary} from 'sentry/components/group/groupSummary';
+import TimeSince from 'sentry/components/timeSince';
+import {EventOrGroupType} from 'sentry/types/event';
+import {
+  type Group,
+  GroupStatus,
+  type GroupUnresolved,
+  IssueCategory,
+  IssueType,
+  PriorityLevel,
+} from 'sentry/types/group';
+
+const unresolvedGroup: GroupUnresolved = {
+  activity: [],
+  annotations: [],
+  assignedTo: null,
+  count: '327482',
+  culprit: 'fetchData(app/components/group/suggestedOwners/suggestedOwners)',
+  firstSeen: '2019-04-05T19:44:05.963Z',
+  filtered: null,
+  hasSeen: false,
+  id: '1',
+  isBookmarked: false,
+  isPublic: false,
+  isSubscribed: false,
+  isUnhandled: false,
+  issueCategory: IssueCategory.ERROR,
+  issueType: IssueType.ERROR,
+  lastSeen: '2019-04-11T01:08:59Z',
+  level: 'warning',
+  logger: null,
+  metadata: {function: 'fetchData', type: 'RequestError'},
+  numComments: 0,
+  participants: [],
+  permalink: 'https://foo.io/organizations/foo/issues/1234/',
+  platform: 'javascript',
+  pluginActions: [],
+  pluginContexts: [],
+  pluginIssues: [],
+  priority: PriorityLevel.MEDIUM,
+  priorityLockedAt: null,
+  project: {
+    platform: 'javascript',
+  },
+  seenBy: [],
+  shareId: '',
+  shortId: 'JAVASCRIPT-6QS',
+  stats: {
+    '24h': [
+      [1517281200, 2],
+      [1517310000, 1],
+    ],
+    '30d': [
+      [1514764800, 1],
+      [1515024000, 122],
+    ],
+  },
+  status: GroupStatus.UNRESOLVED,
+  statusDetails: {},
+  subscriptionDetails: null,
+  title: 'RequestError: GET /issues/ 404',
+  type: EventOrGroupType.ERROR,
+  userCount: 35097,
+  userReportCount: 0,
+};
+
+function IssuesList() {
+  const [activeRowKey, setActiveRowKey] = useState<number | undefined>(undefined);
+
+  const columnOrder: Array<GridColumnOrder<keyof Group>> = [
+    {key: 'title', name: 'Issue'},
+    {key: 'lastSeen', name: 'Age'},
+    {key: 'assignedTo', name: 'Assignee'},
+  ];
```

Comment:
Not sure how to add the Trend column since it's not a `keyof Group`. Was thinking about adding the column later since we don't have the charts yet anyways

Reply:
yep, seems fine to punt on!